probe.go

```diff
@@ -1011,6 +1011,127 @@ func probeCertificates(c FortiHTTP) ([]prometheus.Metric, bool) {
+	}
+
+	// Consider implementing pagination to remove this limit of 1000 entries
+	var rs []LoadBalanceResponse
+	if err := c.Get("api/v2/monitor/load-balance", "vdom=*&start=0&count=1000", &rs); err != nil {
+		log.Printf("Error: %v", err)
+		return nil, false
+	}
+
+	m := []prometheus.Metric{}
+
+	for _, r := range rs {
+		for _, virtualServer := range r.Results {
+			m = append(m, prometheus.MustNewConstMetric(virtualServerInfo, prometheus.GaugeValue, 1, r.VDOM, virtualServer.Name, virtualServer.IP, fmt.Sprintf("%d", virtualServer.Port), virtualServer.Type))
+
+			for _, realServer := range virtualServer.RealServers {
+				realServerModeActive, realServerModeStandby, realServerModeDisabled := 0.0, 0.0, 0.0
+				switch realServer.Mode {
+				case "active":
+					realServerModeActive = 1.0
+					break
+				case "standby":
+					realServerModeStandby = 1.0
+					break
+				case "disabled":
+					realServerModeDisabled = 1.0
+					break
+				}
+
+				realServerStatusUp, realServerStatusDown, realServerStatusUnknown := 0.0, 0.0, 0.0
+				switch realServer.Status {
+				case "up":
+					realServerStatusUp = 1.0
+					break
+				case "down":
+					realServerStatusDown = 1.0
+					break
+				default:
+					realServerStatusUnknown = 1.0
+				}
+
+				m = append(m, prometheus.MustNewConstMetric(realServerInfo, prometheus.GaugeValue, 1, r.VDOM, virtualServer.Name, fmt.Sprintf("%d", realServer.ID), realServer.IP, fmt.Sprintf("%d", realServer.Port)))
+				m = append(m, prometheus.MustNewConstMetric(realServerMode, prometheus.GaugeValue, realServerModeActive, r.VDOM, virtualServer.Name, fmt.Sprintf("%d", realServer.ID), "active"))
+				m = append(m, prometheus.MustNewConstMetric(realServerMode, prometheus.GaugeValue, realServerModeStandby, r.VDOM, virtualServer.Name, fmt.Sprintf("%d", realServer.ID), "standby"))
+				m = append(m, prometheus.MustNewConstMetric(realServerMode, prometheus.GaugeValue, realServerModeDisabled, r.VDOM, virtualServer.Name, fmt.Sprintf("%d", realServer.ID), "disabled"))
+				m = append(m, prometheus.MustNewConstMetric(realServerStatus, prometheus.GaugeValue, realServerStatusUp, r.VDOM, virtualServer.Name, fmt.Sprintf("%d", realServer.ID), "up"))
+				m = append(m, prometheus.MustNewConstMetric(realServerStatus, prometheus.GaugeValue, realServerStatusDown, r.VDOM, virtualServer.Name, fmt.Sprintf("%d", realServer.ID), "down"))
+				m = append(m, prometheus.MustNewConstMetric(realServerStatus, prometheus.GaugeValue, realServerStatusUnknown, r.VDOM, virtualServer.Name, fmt.Sprintf("%d", realServer.ID), "unknown"))
+				m = append(m, prometheus.MustNewConstMetric(realServerSessions, prometheus.GaugeValue, float64(realServer.ActiveSessions), r.VDOM, virtualServer.Name, fmt.Sprintf("%d", realServer.ID)))
+				m = append(m, prometheus.MustNewConstMetric(realServerBytesProcessed, prometheus.CounterValue, float64(realServer.BytesProcessed), r.VDOM, virtualServer.Name, fmt.Sprintf("%d", realServer.ID)))
```

Comment:
Please use `strconv.Itoa()` instead of `printf` and remove the float conversion as with the above changes it will not be necessary anymore
```suggestion
				m = append(m, prometheus.MustNewConstMetric(realServerBytesProcessed, prometheus.CounterValue, realServer.BytesProcessed, r.VDOM, virtualServer.Name, strconv.Itoa(realServer.ID)))
```